src/cli/cli.cpp

```diff
@@ -353,6 +354,28 @@ void Interpreter::ProcessCounters(int argc, char *argv[])
     }
 }
 
+void Interpreter::ProcessDiscover(int argc, char *argv[])
+{
+    ThreadError error = kThreadError_None;
+    uint32_t scanChannels = 0;
+    long value;
+
+    if (argc > 0)
+    {
+        SuccessOrExit(error = ParseLong(argv[0], value));
```

Comment:
The API allows passing a channel mask.  The CLI only allows a specific channel for simplicity.